qiskit_optimization/runtime/qaoa_client.py

```diff
@@ -87,6 +115,70 @@ def __init__(
         self._initial_state = initial_state
         self._mixer = mixer
         self._reps = reps
+        self._use_swap_strategies = use_swap_strategies
+        self._use_initial_mapping = use_initial_mapping
+        self._use_pulse_efficient = use_pulse_efficient
+        self._alpha = alpha
+        self._program_id = "qaoa"
+
+        # Use the setter to check consistency with other settings.
+        self._optimization_level = None
```

Comment:
It looks redundant because you call the setter in the next line.
```suggestion
```

Reply:
The private attribute must usually be set in the initializer before calling the setter, otherwise pylint will complain 🙂

Reply:
Done here: 2f5a09b